osu.Game.Rulesets.Tau/osu.Game.Rulesets.Tau.csproj

```diff
@@ -11,13 +11,10 @@
     <AssemblyName>osu.Game.Rulesets.Tau</AssemblyName>
   </PropertyGroup>
   <ItemGroup>
-    <EmbeddedResource Include="Resources\**"/>
+    <Folder Include="Resources\Samples\Gameplay" />
+    <Folder Include="Resources\Textures" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="ppy.osu.Game" Version="2020.923.0"/>
-  </ItemGroup>
-  <ItemGroup>
-    <Folder Include="Resources\Samples\Gameplay"/>
-    <Folder Include="Resources\Textures"/>
```

Comment:
I'd like to keep the format here the same.

Reply:
This was somehow modified by Visual Studio while updating the osu! NuGet package, will try and get the format back again.